src/IssueViz.Security/OpenInIDE/OpenInIDEResources.resx

```diff
@@ -118,18 +118,30 @@
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
   <data name="Factory_CheckingPort" xml:space="preserve">
-    <value>Checking availability of port {0}</value>
+    <value>[Open in IDE] Checking availability of port {0}</value>
```

Comment:
Miscellaneous: I've added a common prefix to the open in IDE messages to see if that makes the output easier to read when they are mixed up with errors from other sources.